photon-api/src/integTest/scala/com/linkedin/photon/ml/normalization/NormalizationTest.scala

```diff
@@ -21,14 +21,13 @@ import org.apache.spark.rdd.RDD
 import org.testng.Assert._
 import org.testng.annotations.Test
 
-import com.linkedin.photon.ml.{ModelTraining, TaskType}
 import com.linkedin.photon.ml.data.LabeledPoint
 import com.linkedin.photon.ml.model.Coefficients
 import com.linkedin.photon.ml.optimization.{L2RegularizationContext, OptimizerType}
+import com.linkedin.photon.ml.stat.BasicStatisticalSummary
 import com.linkedin.photon.ml.supervised.classification.{BinaryClassifier, LogisticRegressionModel}
 import com.linkedin.photon.ml.test.SparkTestUtils
-
-import com.linkedin.photon.ml.stat.BasicStatisticalSummary
+import com.linkedin.photon.ml.{ModelTraining, TaskType}
```

Comment:
ASCII code 7B `{` comes after all the letters of the alphabet :-/ I just ran Optimize Imports in IDEA on the whole project, with the Spark CodeStyle - let's leave it like that.